xml/decoders/esu/v5standardCVs.xml

```diff
@@ -137,9 +137,9 @@
     <variable CV="111" default="0" item="Motor Option 1">
         <decVal/>
         <label>Gearbox Backlash Compensation</label>
-        <label xml:lang="de">Getriebelastausgleich</label>
+        <label xml:lang="de">Getriebelastausgleich (CV111)</label>
```

Comment:
Why have you added the CV number to some of these items? If I remember correctly, you can see the CV number by hovering other the item with the mouse.

Reply:
I know I can see the CV number when the mouse points to them, but the hint disappears after a few seconds. It is very helpful to see the CV numbers, especially for CVs that are not often used and especially if you have to read the decoder manual.